doc/specifications/rest/REST-API-V2.rst

```diff
@@ -111,7 +121,33 @@ In this document, for the sake of readability, no prefix is used in the URIs. In
 prefixes all REST hrefs.
 
 Remember that URIs to REST resources should never be generated manually, but obtained from earlier REST
- calls.
+calls.
+
+Alias URIs
+----------
+
+By default entity URIs are constructed using a (server local) URL template and
+the database ID of the referenced entity. While this mechanism might be subject
+to change (e.g. to support multi-server/-domain setups with interlinking)
+users are requesting more intuitive ways to identify entities to reference
+entities while creating/updating related entities. Alias URIs help to achieve
+this by providing more human readable resource identifiers. These URIs only
+support GET/HEAD operations and will return a redirect to the cannonical entity
+URL. However, when using them as a reference in a POST/PUT/PATCH request, the
+server will accept them and automatically reference the correct entity.
+
+A request/response cycle for an alias URI is supposed to be executed as follows:
+
+:Resource: /content/types/_by_identifier/article
```

Comment:
Ah, good that you pointed me to this one. Did not see it, yet. I'll dig into how this is realized, because spec is not complete here (missing media types, etc.).

In general, `?something=anything` is a filter on a list, so that might return 0, 1 or even more results, which does not make that a reliable source for referencing a unique item in a link. 

But, let's see how it is implemented anyway …